From a2c3f1bca4bf21371d041d1de589039aa3459e91 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Fri, 26 Feb 2021 17:57:47 +0100
Subject: [PATCH 1/1] cmd/efidebug: add firmware management protocol GUID

Add missing GUID short text used in the efidebug tables and efidebug dh
sub-commands.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 cmd/efidebug.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/cmd/efidebug.c b/cmd/efidebug.c
index bbbcb0a546..e4030f514a 100644
--- a/cmd/efidebug.c
+++ b/cmd/efidebug.c
@@ -507,6 +507,10 @@ static const struct {
 		"System Partition",
 		PARTITION_SYSTEM_GUID
 	},
+	{
+		"Firmware Management",
+		EFI_FIRMWARE_MANAGEMENT_PROTOCOL_GUID
+	},
 	/* Configuration table GUIDs */
 	{
 		"ACPI table",
-- 
2.30.0

